### DISCUSSION TITLE
Wyze Pan Cam v1 firmware md5 mismatch

### DISCUSSION THREAD

**Discussion by MrButlerTron** [gtxaspec/wz_mini_hacks#728] [Q&A]
I have a wyze pan cam (v1 I believe) on the latest firmware 4.10.9.3006 
I cloned the repo, installed 'u-boot-tools`, went in to the v2_install folder, ran `./compile_image.sh pan` 
I keep getting `md5sum failed check, please supply a supported demo.zip file` 
I tried to manually grab `demo_4.10.9.2351.bin.zip` and move that to `demo.zip` in the folder and I got the same error.

Is there a known working version of do folks have other suggestions I can try?

My primary goal is get ethernet connectivity for the camera so I don't have to use WiFi .  I already have a working USB-Micro adapter that takes ethernet and power.  I can see the link come up so that seems like its probably a good model.

Thanks in advance for any help or suggestions.

**Comment by 0mggo** [comment]
Even though wiki says that the [latest firmware is supported](https://github.com/gtxaspec/wz_mini_hacks/wiki/Firmware-Support) for the Wyze Pan Cam v1. The script actually supports [these versions](https://github.com/gtxaspec/wz_mini_hacks/blob/1e1383a78f2d08c0a4425f208f72de988c43d560/v2_install/compile_image.sh#L50-L61). I could not get wifi working with any version though, similar to https://github.com/gtxaspec/wz_mini_hacks/issues/609 which is apparently needed to enable ethernet support.
